changelogs/fragments/2246-terraform.yaml

```
@@ -0,0 +1,4 @@
+minor_changes:
+  - terraform - add a function which checks the used terraform version (https://github.com/ansible-collections/community.general/pull/2246).
```

Function behaviour has been changed.
```suggestion
  - terraform - add a function which retrieves the used terraform version (https://github.com/ansible-collections/community.general/pull/2246).
```